Makefile

```diff
@@ -25,8 +25,9 @@ help:
 	@echo "  migrate                     run the kinto migrations"
 	@echo "  lint                        run the code linters"
 	@echo "  tests                       run all the tests with all the supported python interpreters (same as CI)"
+	@echo "  test-deps                   pulls and runs docker postgres and memcached containers for tests"
```

Comment:
Should this be mentioned in contribution docs?

Reply:
Making a small change to community.rst, but we probably need to give everything a once over. I saw some dated communication channels in there.